platform/core/src/services/UIModalService/index.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * A UI Element
+ *
+ * @typedef {HTMLElement} Modal
+ */
+
+/**
+ * UI Modal
+ *
+ * @typedef {Object} ModalProps
+ * @property {string} [backdrop=false] -
+ * @property {string} [keyboard=false] -
+ * @property {number} [show=true] -
+ * @property {string} [closeButton=true] -
+ * @property {string} [title=null] - 'Modal Title'
+ * @property {boolean} [customClassName=null] - '.ModalClass'
+ */
+
+const uiModalServicePublicApi = {
+  name: 'UIModalService',
```

Comment:
If we're adding a `name`, how do we see it being used?
Is it worth allowing it to be overridden in `createUiModalService`?
Should we add this functionality to our other UI service?

Reply:
The other one is also using the name property, this property was actually used inside the service manager when registering services. We can definitely allow it to be overridden, I'm not sure why... do you have something in mind?